docs/mms_on_fargate.md

```diff
@@ -0,0 +1,341 @@
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.2+SG+Use+existing.png)
+
+5. Now you can connect your service to the LB that was created in the previous section. Select the "Application Load Balancer" and set the LB name:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.3+Add+load+balancing.png)
+
+6. Now you need to specify which port on the LB our service should be listening on:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.4+Configure+load+blancer.png)
+
+7. You are not going to use service discovery now, so uncheck it:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.5+Next.png)
+
+8. In this document, we are not using auto-scaling options. For an actual production system, it is advisable to have this configuration setup.
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.3+Auto+scaling.png)
+
+9. Now you are `done-done-done` creating a running service. You can move to the final chapter of the journey, which is testing the service you created. 
+
+## Test The Inference
+
+First find the DNS name of your LB. It should be in `AWS Console -> Service -> EC2 -> Load Balancers` and click on the LB that you created.
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/lb_dns.png)
+
+Now you can run the health checks using this load-balancer public DNS name, to verify that your newly created service is working:
+
+```bash
+curl InfraLb-1624382880.us-east-1.elb.amazonaws.com/ping 
+```
+
+```text
+http://infralb-1624382880.us-east-1.elb.amazonaws.com/ping
+{
+    "health": "healthy!"
+}
+```
+
+And now you are finally ready to run our inference! Let's download an example image:
+```bash
+curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
+```
+
+The image:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/kitten.jpg)
+
+The output of this query would be as follows,
+
+```bash
+curl -X POST InfraLb-1624382880.us-east-1.elb.amazonaws.com/squeezenet/predict -F "data=@kitten.jpg"
+```
+
+```text
+{
+      "prediction": [
+    [
+      {
+        "class": "n02124075 Egyptian cat",
+        "probability": 0.8515275120735168
+      },
+      {
+        "class": "n02123045 tabby, tabby cat",
+        "probability": 0.09674164652824402
+      },
+      {
+        "class": "n02123159 tiger cat",
+        "probability": 0.03909163549542427
+      },
+      {
+        "class": "n02128385 leopard, Panthera pardus",
+        "probability": 0.006105933338403702
+      },
+      {
+        "class": "n02127052 lynx, catamount",
+        "probability": 0.003104303264990449
+      }
+    ]
+  ]
+}
+```
+
+## Customize the containers to server your custom deep learning models
+
+For this section, we will show you how you could wrap the official container images with your custom configuration. 
```

Comment:
could --> can

your --> a

official meaning the ones that are provided by MMS here? 
maybe just use "provided" instead...

Reply:
Done